package.json

```
@@ -18,6 +18,7 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "npm run clean && tsc && cp -r templates dist/templates && cp -r examples dist/examples",
+    "publish-to-npm": "./cleanup_for_publish.sh && npm install && npm run build && npm publish",
```

a crucial step is to run `npm publish --dry-run` before running npm publish


```suggestion
    "publish-to-npm": "./cleanup_for_publish.sh && npm install && npm run build",
```